NwPluginAPI/Enums/ServerEventType.cs

```diff
@@ -334,13 +342,13 @@ public enum ServerEventType : int
         /// </remarks>
         PlayerPreauth = 39,
 
-        /// <summary>
-        /// Executed when player receives effect.
-        /// </summary>
-        /// <remarks>
-        /// Parameters: <see cref="IPlayer"/> player, <see cref="PlayerEffect"/> effect.
-        /// </remarks>
-        PlayerReceiveEffect = 40,
+		/// <summary>
+		/// Executed when player receives effect.
+		/// </summary>
+		/// <remarks>
+		/// Parameters: <see cref="IPlayer"/> player, <see cref="PlayerEffect"/> effect, <see cref="byte"/> intensity, <see cref="float"/> duration.
```

Comment:
Duration is set independently and may be (and even is) set after this event is called. That would require a second event - duration changed.

Reply:
My change was to add Intensity, Duration already existed.

Reply:
![image](https://user-images.githubusercontent.com/36207738/209983077-e9caf723-dea2-4654-956b-918843da8012.png)

Reply:
Ah, my bad. Adding intensity is possible.